### DISCUSSION TITLE
Welcome to AviSynthPlus Discussions!

### DISCUSSION THREAD

**Discussion by qyot27** [AviSynth/AviSynthPlus#201] [General]
This seems to be a new feature GitHub has rolled out, and I figure it'll be useful to have it enabled for stuff that doesn't quite fit into Issues or the Wiki.  I'm sure that not everyone that potentially wants to interact with upstream AviSynth+ in some way will necessarily have/use an account on Doom9, so having a place right here in the repo to do it is probably a good idea.

**Comment by pinterf** [comment]
Welcome!
